web/pageComponents/topicPages/KeyNumbers/KeyNumberItem.tsx

```diff
@@ -10,7 +10,7 @@ const Wrapper = styled.div`
   min-width: var(--card-maxWidth);
   flex-basis: 100%;
   @media (min-width: 750px) {
-    flex-basis: 45%;
```

Comment:
you are mixing flex props such as "flex-wrap" with grid... you can make the code a lot better following the grid practices..

You should be able to achieve a nice looking grid just by modifying the parent wrapper in keyNumbers.tsx:

```
const Container = styled.div`
  display: grid;
  gap: var(--space-large);
  margin-bottom: var(--space-large);
  grid-template-columns: repeat(1, 1fr);
  @media (min-width: 768px) {
    grid-template-columns: repeat(2, 1fr);
  }
  @media (min-width: 1024px) {
    grid-template-columns: repeat(3, 1fr);
  }
`
```

and with that you can clean up all the hack solutions here in the child.

Reply:
The fragment on line 22 is also useless since there is already a wrapper, so it can be cleaned up too

Reply:
Done